frontend/src/app/commonComponents/Header.jsx

```diff
@@ -1,6 +1,12 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { Link, NavLink } from "react-router-dom";
+import {
+  PATIENT_TERM,
+  PATIENT_TERM_CAP,
+  PATIENT_TERM_PLURAL,
+  PATIENT_TERM_PLURAL_CAP,
```

Comment:
Only PATIENT_TERM_CAP is used so the others should be removed. If you have ESLint configured, it should put a squiggly under them to let you know.